### PR TITLE
chore: enable 2fa in qa

### PR DESCRIPTION
**Pull request by Zangetsu101**

### CODE REVIEW

File: infrastructure/docker-compose.production-deploy.yml [hunk 1]
```diff
@@ -90,6 +90,7 @@ services:
 
   scheduler:
     environment:
+      - QA_ENV=true
```
Comment (rikukissa) [inline infrastructure/docker-compose.production-deploy.yml:93]: What's this for?

Comment (Zangetsu101) [reply infrastructure/docker-compose.production-deploy.yml:93]: when I was taking a look at all the places that had QA_ENV variable, I noticed that staging and production had it missing from scheduler so I added it in

Comment (Zangetsu101) [reply infrastructure/docker-compose.production-deploy.yml:93]: scheduler is one of the services that actually uses this flag, along with auth, gateway, metrics, config & user-mgnt

Comment (Zangetsu101) [reply infrastructure/docker-compose.production-deploy.yml:93]: it's still missing from gateway :thinking: Should I add that in too I wonder

**Pull request merged**